.github/workflows/release.yaml

```
@@ -44,14 +44,14 @@ jobs:
 
     - name: Output branch name
       id: branch
-      run: echo "::set-output name=branch_name::$BRANCH_NAME"
+      run: echo "{branch_name}={$BRANCH_NAME}" >> $GITHUB_OUTPUT
```

These braces are part of the GitHub example only and shouldn't be included. I made the same mistake myself.
```suggestion
      run: echo "branch_name=$BRANCH_NAME" >> $GITHUB_OUTPUT
```